README.md

```diff
@@ -1,4 +1,25 @@
-# validator-smart-contracts
+# Validator Smart Contracts
```

Comment:
The readmes in this repo are pretty short. And given that sub-directory readme discoverability is pretty low, I would argue that merging all readmes into one file would make this repo easier to understand.

Reply:
Might do that in another PR!